ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh

```diff
@@ -83,6 +83,11 @@ cat << EOF > ~/fetch-kubeconfig.yml
       regexp: '    server: https://api.cnfdc5.t5g.lab.eng.bos.redhat.com:6443'
       line: "    server: https://${TEST_CLUSTER_API_IP}:6443"
     delegate_to: localhost
+  - name: Grab the CNF tests skip list
+    fetch:
+      src: /home/kni/telco5g-cnf-tests-skip-list.txt
```

Comment:
y not keep this list in github?

Reply:
do you mean here in our telco5g dir? i tried that but got this error, so i thought we can't keep txt files here
`time="2022-08-22T14:14:30Z" level=fatal msg="failed to parse arguments" error="failed to load registry: invalid file name: /ci-operator/step-registry/telco5g/cnf-tests/telco5g-cnf-tests-skip-list.txt"` 
maybe we can move to another format for that, json probably. if you think it's better to keep it here.

Reply:
We can "cat" this file to $SHARED_DIR and keep it there, as we do with all configs here.